lib/webhook.js

```diff
@@ -21,9 +21,7 @@ module.exports = (bot, opt) => {
 
   // Check port
   if (!PORTS.includes(port)) {
-    let error = `[bot.error.webhook] allowed ports: ${ PORTS.join(', ') }`;
-    this.event('error', { error });
```

Comment:
Nah, I haven't tested your last pull request. Instead of `this.event('error', { error });` must be `bot.event('error', { error });`, *this* don't work in arrow function. Every time you pass "wrong" port - error occurred and passes to main *telebot.js* error handler.